### PR TITLE
msg/vehicle_global_position: remove velocity fields (duplicates of local vx, vy, vz)

### PR DESCRIPTION
**Pull request by dagar**

### CODE REVIEW

File: src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp [hunk 1]
```diff
@@ -299,29 +299,29 @@ AttitudeEstimatorQ::Run()
 			}
 		}
 
-		if (_global_pos_sub.updated()) {
-			vehicle_global_position_s gpos;
+		if (_gps_sub.updated()) {
```
Comment (bresch) [inline src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp]: `gps.eph` is in general not the same as `global_pos.eph`

Comment (dagar) [reply src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp]: For gating the mag declination update it seems fine (table is extremely coarse). Using the GPS velocity data seems sketchy though. Realistically we're not going to spend much time verifying attq, so I'll move the velocity change to local position.

Comment (dagar) [reply src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp]: Done.

File: src/examples/bottle_drop/bottle_drop.cpp [hunk 2]
```diff
@@ -176,11 +178,7 @@ BottleDrop::BottleDrop() :
 	_mavlink_log_pub(nullptr),
 	_command_sub(-1),
 	_wind_estimate_sub(-1),
-	_command {},
-	_global_pos {},
-	ref {},
 	_actuator_pub(nullptr),
```
Comment (bresch) [inline src/examples/bottle_drop/bottle_drop.cpp]: you already initialized `_actuator_pub` above

**Pull request merged**